services/tenant-ui/.eslintrc

```diff
@@ -4,6 +4,9 @@
   "plugins": [
     "@typescript-eslint"
   ],
+  "rules": {
+    "@typescript-eslint/no-explicit-any": "off",
```

Comment:
Had to add this because middleware injects new fields into REquest (common thing) and then TS doesn't like that... I'm sure there's a standard way to handle that we can look up.